ui-charts/src/manchette/hooks/useManchetteWithSpaceTimeChart.ts

```diff
@@ -58,6 +70,14 @@ type State = {
   scales: SpaceScale[];
 };
 
+export type SplitPoint<Payload = unknown> = {
+  position: number;
+  size: number;
```

Comment:
Nit: would be nice to rename this to `height` to make it clearer it's the vertical size, and add a `// in pixels` comment.

Reply:
I kept `size` rather than `height` as I imagined we will allow swapping time and space axis at some point in the future. It's already implemented [in the GET](https://openrailassociation.github.io/osrd-ui/?path=/story/spacetimechart-split--default&args=swapAxis:!true) because it was in the requirements, but I haven't seen that feature in any OSRD designs yet...

Reply:
Makes sense to me.